docs/source/core_versions.rst

```diff
@@ -23,9 +23,7 @@ The tuple identify which sets of parameters have been verified
 by OpenHW Group, and once RTL Freeze is achieved, no further
 non-logically equivalent changes are allowed on that set of parameters.
 
-The RTL Freeze version of the core is indentified by a GitHub
-tag with the format cv32e41p_vMAJOR.MINOR.PATCH (e.g. cv32e41p_v1.0.0).
-In addition, the release date is reported in the documentation.
+**The core RTL is not yet frozen.**
```

Comment:
can we add a single comment saying that the RV32IMC ISA subset has been kept Sequentially Equivalent to the CV32E40P (that has been fully verified of that subset) to keep the health of the CPU stable? or something like this

Reply:
thanks @tariqkurd